docs-source/content/userguide/istio/istio.md

```diff
@@ -0,0 +1,275 @@
+
+#### Using the operator with Istio support
+
+{{% notice note %}}
+These instructions assume that you are using a Kubernetes cluster with
+[Istio Installation](https://istio.io/latest/docs/setup/install/) installed and configured already.  The operator will not install
+Istio for you.
+{{% /notice %}}
+
+You can deploy the operator into a namespace which has Istio automatic sidecar
+injection enabled.  Before installing the operator, create the namespace you
+wish to run the operator in, and label it for automatic injection.
+
+```
+$ kubectl create namespace weblogic-operator
+$ kubectl label namespace weblogic-operator istio-injection=enabled
+```
+
+After the namespace is labeled, you can install the operator using the normal
+method.  When the operator pod starts, you will notice that Istio automatically
+injects an `initContainer` called `istio-init` and the envoy container `istio-proxy`.
+
+You can check this using the following commands:
+
+```
+$ kubectl --namespace weblogic-operator get pods
+$ kubectl --namespace weblogic-operator get pod weblogic-operator-xxx-xxx -o yaml
+```
+
+In the second command, change `weblogic-operator-xxx-xxx` to the name of your pod.
+
+#### Creating a domain with Istio support
+
+You can configure your domains to run with Istio automatic sidecar injection enabled.
+Before creating your domain, create the namespace you wish to run the domain in,
+and label it for automatic injection.
+
+```
+$ kubectl create namespace domain1
+$ kubectl label namespace domain1 istio-injection=enabled
+```
+
+To enable the Istio support for a domain, you need to add the
+`configuration` section to your domain custom resource YAML file as shown in the
+following example:  
+
+```
+apiVersion: "weblogic.oracle/v7"
+kind: Domain
+metadata:
+  name: domain2
+  namespace: domain1
+  labels:
+    weblogic.resourceVersion: domain-v2
+    weblogic.domainUID: domain2
+spec:
+  ... other content ...
+  configuration:
+    istio:
+      enabled: true
+      readinessPort: 8888
+```
+
+To enable the Istio support, you must include the `istio` section
+and you must set `enabled: true` as shown.  The `readniessPort` is optional
+and defaults to `8888` if not provided, it is used for readiness health check.
+
+##### How Istio-enabled domains differ from regular domains
+
+Istio enforces a number of requirements on Pods.  When you enable Istio support in the domain resource, the
+introspector job automatically creates configuration overrides with the necessary channels for the domain to satisfy Istio's requirements, including:
+
+When deploying a domain with Istio sidecar injection enabled.  The WebLogic Server Kubernetes Operator automatically add the following network
+channels via configuration overrides.
+
+https://istio.io/latest/docs/ops/configuration/traffic-management/protocol-selection/
+
+For non-SSL traffic:
+
+|Name|Port|Protocol|Exposed as a container port|
```

Comment:
In this table and the following one, all the entries in the Name and Protocol columns should be in `code font`, that means surrounded by back ticks. For example, http-probe -> `http-probe`

Reply:
i don't think the content in the second column should be backticked

Reply:
or the last one.. and "Yes"/"No" is probably better than "Y"/"N"